### DISCUSSION TITLE
SFTP chmod dialog

### DISCUSSION THREAD

**Discussion by arseniigorkin** [isontheline/pro.webssh.net#661] [Ideas]
Hi!
Thank you for a great app!

It would be nice if there would be added a dialog to setting up file/dir permissions (such as chmod in terminal) in SFTP.

Thank you!

**Comment by isontheline** [comment]
Hello @arseniigorkin 👋

That could be great you're right!

I'm opening an issue in order to track implementation : https://github.com/isontheline/pro.webssh.net/issues/662
